var/spack/repos/builtin/packages/intel-daal/package.py

```diff
@@ -48,11 +41,12 @@ class IntelDaal(IntelInstaller):
 
     provides('daal')
 
-    def install(self, spec, prefix):
-
-        self.intel_prefix = os.path.join(prefix, "pkg")
```

Comment:
@lee218llnl Any reason we were installing to `<prefix>/pkg` instead of `<prefix>`?